pages/SingleArticlePage.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import { Link, useNavigate, useParams } from "react-router-dom";
```

Comment:
Do you need link and useNavigate here? Unused variables can cause issues in hosting as they can be treated as errors.